content/2.components/scratch-to-reveal.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Scratch To Reveal
+description: The ScratchToReveal component creates an interactive scratch-off effect with customizable dimensions and animations, revealing hidden content beneath.
+navBadges:
+  - value: New
+    type: lime
+---
+
+::ComponentLoader{label="Preview" componentName="ScratchToRevealDemo" type="examples" id="scratch-to-reveal"}
+::
+
+## API
+
+| Prop Name              | Type                     | Default | Description                                                                                   |
+| ---------------------- | ------------------------ | ------- | --------------------------------------------------------------------------------------------- |
+| `class`                | `string`                 | `""`    | The class name to apply to the component.                                                     |
+| `width`                | `number`                 | `""`    | Width of the scratch container.                                                               |
+| `height`               | `number`                 | `""`    | Height of the scratch container.                                                              |
+| `minScratchPercentage` | `number`                 | `50`    | Minimum percentage of scratched area to be considered as completed (Value between 0 and 100). |
+| `gradientColors`       | `[string,string,string]` | `-`     | Gradient colors for the scratch effect.                                                       |
+
+| Event Name | Payload | Description                                        |
+| ---------- | ------- | -------------------------------------------------- |
+| `compete`  | `-`     | Callback function called when scratch is completed |
+
+| Slot Name | Default Content | Description                            |
+| --------- | --------------- | -------------------------------------- |
+| `default` | `-`             | The content to display in the marquee. |
```

Comment:
What is `marquee` here?

Reply:
🤣It is this in the source document. Maybe Change it to 'The text below the scratch-off ticket'?

Reply:
@Whbbit1999 Yes please update this.